### PR TITLE
NFV-HCI minor fixes

### PR DESCRIPTION
**Pull request by eshulman2**
minor fixes for NFV hci deployment

### CODE REVIEW

File: automation/vars/default.yaml [hunk 1]
```diff
@@ -271,8 +271,10 @@ vas:
         post_stage_run:
           - name: Deploy Ceph
             type: playbook
-            source: "../../playbooks/ceph.yml"
+            source: "../../hooks/playbooks/ceph-bm.yml"
```
Comment (abays) [inline automation/vars/default.yaml:274]: I don't see this listed here: https://github.com/openstack-k8s-operators/ci-framework/tree/main/hooks/playbooks

Comment (eshulman2) [reply automation/vars/default.yaml:274]: Hi Andrew it is in this patch https://github.com/openstack-k8s-operators/ci-framework/pull/2246/files that we are working on getting merged

**Comment by abays** [comment]
@eshulman2 Has this been tested?

**Comment by eshulman2** [comment]
Hi @abays yes it was tested here: https://sf.hosted.upshift.rdu2.redhat.com/zuul/t/tripleo-ci-internal/build/549e1754d9e140ba9baabbf0fa9804e3 the job failed on some tests but unrelated to architecture, just added a small thing for consistency and going to re-run now to verify. will ping you after the job finishes.

**Review by abays — APPROVED** [review]
We'll need to massage the changes to `default.yaml` into https://github.com/openstack-k8s-operators/architecture/pull/383, as the file is being dropped there and moved elsewhere

/lgtm
/approve

**Comment by eshulman2** [comment]
@abays I copied the changes between the default file and the file that should be used after that patch to avoid issues

**Comment by cjeanner** [comment]
/lgtm
re-adding the label; Gate failed because there was no mock file for this scenario, meaning the cifmw_baremetal_hosts wasn't set.

**Comment by cjeanner** [comment]
recheck-gate

**Pull request merged**